modules/virt-single-node-cluster.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-single-node-cluster_{context}"]
+= Changes in {VirtProductName} behavior on a single node cluster
+
+By default, you can use {VirtProductName} with single node clusters, also known as xref:../../operators/operator_sdk/osdk-ha-sno.adoc#osdk-ha-sno[Single Node OpenShift (SNO)]. However, these clusters are not configured for high-availability operation, which results in significant changes to the behavior of the cluster:
+
+* {VirtProductName} components have only one replica, and `PodDisruptionBudgets` are skipped.
```

Comment:
s/`PodDisruptionBudgets`/pod disruption budgets

Per our [API object formatting guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting), you do not need to mark up general references to the object in backticks.